src/kvstore/RocksEngine.h

```diff
@@ -81,12 +80,29 @@ class RocksPrefixIter : public KVIterator {
         return folly::StringPiece(iter_->value().data(), iter_->value().size());
     }
 
-private:
+protected:
     std::unique_ptr<rocksdb::Iterator> iter_;
     rocksdb::Slice prefix_;
 };
 
 
+class RocksRangeWithPrefixIter : public RocksPrefixIter {
+public:
+    RocksRangeWithPrefixIter(rocksdb::Iterator* iter, rocksdb::Slice start, rocksdb::Slice prefix)
+        : RocksPrefixIter(iter, prefix)
+        , start_(start) {}
+
+    ~RocksRangeWithPrefixIter() = default;
+
+    bool valid() const override {
```

Comment:
Why override the method?  It looks the same.

Reply:
Fixed